src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -66,6 +66,33 @@ class BasicWorkflow extends FunSpec with Matchers {
         }
 
         //-----------------------------------------------------------------------------------------------
+        // public symbol search - java.io.File
+
+        val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
+        javaSearchSymbol match {
+          case SymbolSearchResults(List(
+            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
+              Some(_)),
+            TypeSearchResult("java.io.File", "File", 'class,
+              Some(_)),
+            TypeSearchResult("org.ensime.util.FileUtilSpec", "FileUtilSpec", 'class, None))) =>
```

Comment:
I assumed on first reading that it was an expression like j.i.File where the subparts could be camel or partials but all would have to match.
I will raise a new ticket for the count break (I looked, it appears to pass all the way to Lucene)